### DISCUSSION TITLE
New master release

### DISCUSSION THREAD

**Discussion by wgntn** [cdcseacave/openMVS#1003] [Q&A]
Just curious how close we are to getting a new merge into master. Thanks

**Comment by cdcseacave** [comment]
soon, are you interested in a particular feature?

**Reply by EliDavis3D** [reply]
* [The ability to add masks to textures](https://github.com/cdcseacave/openMVS/commit/c69bae4f9813eb9b5a0a9d6e6ae9437c54e59a30)
* [The transfer-texture-file flag](https://github.com/cdcseacave/openMVS/commit/af8578ae5e4a2be804b9959ff678bd0157f5af82) - With the assumption that we can transfer textures to a decimated mesh with this
* [The python api](https://github.com/cdcseacave/openMVS/commit/0c9d9128ba13a7df70ca3a93257255355d533d2c)

**Reply by wgntn** [reply]
This ^^^

**Comment by cdcseacave** [comment]
New release v2.2.0 published
